packages/ui-components/src/components/DataTable/DataTable.tsx

```diff
@@ -62,11 +83,12 @@ export const DataTable = ({ columns, data, className, rowLimit, total }) => {
     <TableContainer className={className}>
       <StyledTable {...getTableProps()} style={{ minWidth: columnsData.length * 140 + 250 }}>
         <TableHead>
-          {headerGroups.map(({ getHeaderGroupProps, headers }) => (
```

Comment:
I get some errors in this section about  implicit any types. I don't mind if you want to add ts-ignore since it's not really possible to type data and columns

Reply:
I'm not sure if possibly something related got added into a different PR, but if you run `install` on the package it should add the types and remove the errors :)

Reply:
Just noting that this is fixed on the 3rd part of the PR [here](https://github.com/beyondessential/tupaia/pull/4579/)